scripts/deno.mjs

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { readdir, mkdir, readFile, writeFile, rm, stat, copyFile } from 'fs/promises';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const baseDirectory = join(__dirname, '..');
+const denoPath = join(baseDirectory, '.deno');
```

Comment:
Unless you insist on using `join`, Kyra would ree at this approach as opposed to 

```suggestion
const baseDirectory = new URL('../', import.meta.url);
const denoPath = new URL('.deno/', baseDirectory);
```

Note that if you still need the regular file path at a later stage you can always call:
```mjs
import { fileURLToPath } from 'url';
fileURLToPath(urlThatIsAFileOrDirectoryHere),
```
However, all `fs/promises` functions take a `PathLike` (from `@types/node/fs.d.ts`):
```ts
export type PathLike = string | Buffer | URL;
```